.codespellignorewords

```diff
@@ -2,3 +2,4 @@ acn
 ACN
 inflight
 thead
+wronly
```

Comment:
where does this occur that we need to ignore it instead of fixing it?

Reply:
It's a part of the file descriptor mode `O_WRONLY`, as in "write only." Example: https://github.com/OpenLightingProject/ola/blob/c9add79a8d1c83ab209146006dae87906803b64f/plugins/opendmx/OpenDmxThread.cpp#L71

Previous versions of codespell just didn't pick it up it seems.

Reply:
aah, O_WRONLY absolutely makes sense. I thought it to be a misspelling of "wrongly"

Reply:
> I thought it to be a misspelling of "wrongly"

Indeed, codespell seemed to think exactly that!

Reply:
Yeah the default codespell word regex doesn't do snake case so doesn't need to ignore O_WRONLY type cases as it just thinks they're one not two words.

We could do the line ignore, but given we only use the word wrongly once, it's probably a bit irrelevant for now.